client-react/src/pages/app/functions/function/function-log/FunctionLogAppInsightsDataLoader.tsx

```diff
@@ -219,7 +228,7 @@ const FunctionLogAppInsightsDataLoader: React.FC<FunctionLogAppInsightsDataLoade
 
   const reconnectQueryLayer = () => {
     const newQueryLayer = new QuickPulseQueryLayer(getQuickPulseQueryEndpoint(), defaultClient);
-    newQueryLayer.setConfiguration([], getDefaultDocumentStreams(), [], functionsRuntimeVersion);
+    newQueryLayer.setConfiguration([], getDefaultDocumentStreams(), [], useNewFunctionLogsApi());
```

Comment:
If using `useMemo` instead of `useCallback`:
```suggestion
    newQueryLayer.setConfiguration([], getDefaultDocumentStreams(), [], useNewFunctionLogsApi);
```